GeneratorInterface/Core/test/test_FailingGeneratorFilter_cfg.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
+import sys
+
+process = cms.Process("TEST")
+
+process.source = cms.Source("EmptySource")
+
+process.maxEvents = cms.untracked.PSet(input=cms.untracked.int32(10))
```

Comment:
Why not simply "10"? (just curious....)

Reply:
In newer releases one can do `process.maxEvents.input = 10` because the `cms.Process` object internally knows that `maxEvents` is a `untracked.PSet` and knows what are the allowed parameters (and their types) with that `PSet`. Unfortunately, 10_6 doesn't have that change so one is required to explicitly use all the types.